docs/source/getting_started/index.rst

```diff
@@ -1,99 +1,123 @@
 
-Step 2
-^^^^^^
+.. code-block:: shell
 
-Step 3
-^^^^^^
+  docker run --name some-postgres \
+  -e POSTGRES_USER=postgres \
+  -e POSTGRES_PASSWORD=mysecretpassword \
+  -p 5432:5432 \
+  --network=iroha-network \
+  -d postgres:9.5
 
-Playing around with settings
-----------------------------
+Creating blockstore
+^^^^^^^^^^^^^^^^^^^
+Before we run Iroha container, we should create persistent volume to store files, storing blocks for the chain. It is done via the following command:
 
+.. code-block:: shell
 
-Step 1
-^^^^^^
+  docker volume create blockstore
 
-Step 2
-^^^^^^
+Configuring Iroha network
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Now we need to configure our Iroha network. This includes creating a configuration file, generating keypairs for a users, writing list of peers and creating a genesis block. However, we have prepared an example configuration for this guide, so you can start playing with Iroha faster. In order to get those files, you need to clone the `Iroha repository <github.com/hyperledger/iroha>`_ from Github.
 
-Step 3
-^^^^^^
+.. code-block:: shell
 
-Running a single instance
--------------------------
+  git clone -b develop https://github.com/hyperledger/iroha
 
+Starting Iroha container
+^^^^^^^^^^^^^^^^^^^^^^^^
+We are ready to launch our Iroha container. Let's do it with the following command
 
-Step 1
-^^^^^^
+.. code-block:: shell
 
-Step 2
-^^^^^^
+  docker run -it --name iroha \
+  -p 50051:50051 \
+  -v $(pwd)/iroha/example:/opt/iroha_data \
+  -v blockstore:/tmp/block_store \
+  --network=iroha-network \
+  --entrypoint=/bin/bash \
+  hyperledger/iroha-docker:develop
 
-Step 3
-^^^^^^
+Launching Iroha Daemon
+^^^^^^^^^^^^^^^^^^^^^^
+Now you are in the interactive shell of Iroha's container. To actually run Iroha, we need to launch Iroha daemon – ``irohad``.
 
-Sending transactions and queries
---------------------------------
+.. code-block:: shell
 
+  irohad --config config.docker --genesis_block genesis.block --keypair_name node0
 
-Step 1
-^^^^^^
+In usual situation, you need to provide config file, generate genesis block and keypair. However, as a part of this guide we provide an example configuration for you. Don't use it in a production though.
 
-Step 2
-^^^^^^
+.. hint:: You can get more information about ``irohad`` and its launch options in this section
 
-Step 3
-^^^^^^
+Congratulations! You have an Iroha node up and running! In the next section we will test it by sending some transactions.
 
-Deploy a network of Iroha peers
--------------------------------
+Interacting with Iroha network
+------------------------------
+You can interact with Iroha using various ways. You can use our client libraries to write code in various programming languages (e.g. Java, Python, Javascript, Swift) which communicates with Iroha. Alternatively, you can use ``iroha-cli`` – our command-line tool for interacting with Iroha. As a part of this guide, let's get familiar with ``iroha-cli``
```

Comment:
can you please put a note that cli that they are going to use was engineered very fast, is not having any tests, and we either need help or in the future we will deliver a better one? 😭